test-framework/src/main/java/org/keycloak/test/framework/util/UIUtils.java

```diff
@@ -0,0 +1,29 @@
+package org.keycloak.test.framework.util;
+import org.apache.commons.lang3.StringUtils;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+
+public final class UIUtils {
+
+    public static final String VALUE_ATTR_NAME = "value";
+
+    public static String getTextInputValue(WebElement input) {
+        return input.getAttribute(VALUE_ATTR_NAME);
+    }
+
+    public static void setTextInputValue(WebElement input, String value) {
+        input.click();
+        input.clear();
+        if (!StringUtils.isEmpty(value)) { // setting new input
```

Comment:
Do we need Apache commons for this? Isn't `value.isEmpty()` enough?

Reply:
Will remove whole class for now